experiments/golden-results/vct-summary.txt

```diff
@@ -3,11 +3,6 @@ Calling function: Process_Declaration
 Error message: Exception declaration
 Nkind: N_Exception_Declaration
 --
-Occurs: 61 times
```

Comment:
Why does this change?

Reply:
That is a bit of a mystery. There is the same thing in #220.

Reply:
Looking at the commit history, it would almost seem that merging 214 and 217 happened at the same time and resulted in a state that would not pass CI.

Reply:
Is it possible that master or other changes recently merged cause this?

Reply:
Well 214 was about ignoring pragma in declarations.

Reply:
OK.  Would it be possible to create a separate PR that brings the golden results up to date w.r.t. master?  Having an idea of what issues each patch fixes is really useful from a PM point of view.

Reply:
Sure: https://github.com/diffblue/gnat2goto/pull/222.

Reply:
Thank you.